src/explorer.ts

```diff
@@ -126,7 +134,6 @@ class KubernetesWorkloadFolder extends KubernetesFolder {
     constructor() {
         super("workload", "Workloads");
     }
-
```

Comment:
Did a blank separator line accidentally get removed here?  Would prefer it to stay I think.

Reply:
Sorry about that, fixed this.